template.json

```diff
@@ -47,6 +47,8 @@
             "build:withLicenses": "craco build",
             "test": "react-scripts test",
             "prettier": "prettier --write '**/*.{js,jsx,ts,tsx,gql,css,json,html}'",
+            "lint:google-fonts": "./scripts/lint-google-fonts.sh",
+            "codegen": "./scripts/swagger-codegen.sh",
             "lint": "eslint 'src/**/*.ts*'"
```

Comment:
One small last change: could you also run the `lint:google-fonts` step when the `lint` step runs? Add it with a  `&&` afterwards.